Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -171,6 +171,12 @@ void MenuBar::AddViewMenu()
   AddTableColumnsMenu(view_menu);
 }
 
+void MenuBar::AddOptionsMenu()
+{
+  QMenu* options_menu = addMenu(tr("Options"));
+  options_menu->addAction(tr("Hotkey Settings"), [this] { emit ConfigureHotkeys(); });
```

Comment:
This can be just `options_menu->addAction(tr("Hotkey Settings"), this, &MenuBar::ConfigureHotkeys);`.